OxfordBodleian/Juel-Jensen/BDLaethf27.xml

```diff
@@ -50,35 +285,144 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <width>101</width>
+                                </dimensions>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d2" type="miniature" corresp="#pic2">
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>118</height>
+                                    <width>101</width>
+                                </dimensions>
+                                one or two miniatures? Now two for ease of encoding below
+                            </decoNote>
+                            
+                                <decoNote xml:id="d3" type="miniature" corresp="#pic3">
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>118</height>
+                                    <width></width>
+                                </dimensions>
+                                one or two miniatures? Now two for ease of encoding below
+                            </decoNote>
+                            
+                            <decoNote xml:id="d4" type="miniature" corresp="#pic4">
+                                <desc></desc>
+                                <dimensions unit="mm">
+                                    <height>75</height>
+                                    <width>98</width>
+                                </dimensions>
+                            </decoNote>
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>The name of the owner, mentioned in supplications throughout, was
+                                        <persName ref="PRS13863WalattaE" role="owner"/>.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc>The manuscript contains no scribal corrections.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <desc>The verso is empty.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1">The three parchment strips are slightly overlapping and sewn together with a running stitch using a narrow parchment strip each.</decoNote>
+                                <decoNote xml:id="b2">A textile strip is attached to a hole in the middle of the lower edge of the scroll.</decoNote>
```

Comment:
It seems attached to the right corner, am I wrong?

Reply:
Very right!